app/api/leaderboard/functions.ts

```diff
@@ -113,3 +113,26 @@ export default async function fetchGitHubReleases(
     )
     .slice(0, sliceLimit);
 }
+
+export async function fetchAllReposName() {
+  const response = await octokit.graphql({
```

Comment:
Use `paginate` to fetch all repo names

```suggestion
  const response = await octokit.graphql.paginate({
```